system-tests/services/_scripts/external-volumes/teardown

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+pip install boto3
+pip install retrying
+
+cmd="./services/_scripts/external-volumes/volclean.py \
+-y -r $REGION -k $ACCESS_KEY_ID -s \"$SECRET_ACCESS_KEY\" \
+--ignore-metrics --tags 'Name:^integration-test-dcos-ui-$TEST_UUID'"
+
+MATCH_STRING="*Removing*Volumes*"
+TIMEOUT=30
+counter=0
+
+num_vols_delete=$NUM_VOLS_DELETE
```

Comment:
Let's just hard code this and write a comment about what it is. Currently we need to clean up 2 volumes because we create 2 for the tests but if we write more tests we'll need to clean up more.